maven-core/src/test/java/org/apache/maven/artifact/resolver/filter/ExclusionArtifactFilterTest.java

```diff
@@ -140,4 +145,15 @@ void testMultipleExclusionsExcludeGroupIdWildcard() {
 
         assertThat(filter.include(artifact), is(false));
     }
+
+    @Test
+    void testExcludeWithGlob() {
+        Exclusion exclusion = new Exclusion();
+        exclusion.setGroupId("*");
```

Comment:
needs tests for ** as well here, or maybe you're not using globstar? Docs on the format are needed here.